.travis.yml

```diff
@@ -14,7 +14,7 @@ language: java
 install: /bin/true
 
 script: |
-  ./mvnw clean verify \
+  ./mvnw clean verify -DskipItTests \
```

Comment:
Should this have said skipITs?

Reply:
skipITs is failsafe built-in properties. I need a property to skip the docker plugin and failsafe together (in a profile). Do you think we should just reuse that properties?

Reply:
I think it would be better for consistency if we just use one property for the idea of skipping properties (across all our projects). Unless there is a reason to distinguish this case...?

Reply:
The only thing might be we might introduce a different set of integration tests that don't involve docker, in which case they can be run in travis or environment that don't have docker installed. Right now, I only created one IT case that uses docker deployment.